torch/_inductor/async_compile.py

```diff
@@ -265,12 +265,16 @@ def task():
 
         return self.submit(task)
 
-    def rocm(self, source_code, dst_file_ext, aot_compile=False):
+    def rocm(
+        self, source_code, dst_file_ext, aot_compile=False, generate_standalone=False
+    ):
         kernel_code_log.info("ROCm Kernel:\n%s", source_code)
 
         def task():
             if aot_compile:
                 _ = ROCmCodeCache.compile(source_code, dst_file_ext="o")
+            if generate_standalone:
+                _ = ROCmCodeCache.compile(source_code, dst_file_ext="exe")
```

Comment:
do you think just referencing `config.rocm.generate_test_runner` here would be cleaner?